xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/src/main/resources/Mail/SendMailStatus.xml

```diff
@@ -76,32 +76,36 @@
 {{translation key="admin.emailstatus.hint" /}}
 
 ##
-## Livetable #
+## Live Data #
 ##
 #set($columns = ['date', 'batchId', 'type', 'recipients', 'state', 'error', 'actions'])
-#set($columnsProperties = {
-    'date' : { 'type' : 'date' },
-    'batchId' : { 'type' : 'text' },
-    'type' : { 'type' : 'text' },
-    'recipients' : { 'type' : 'text' , 'html' : 'true' },
-    'state' : { 'type' : 'text', 'size' : 6 },
-    'error' : { 'type' : 'text' },
-    'actions' : { 'type' : 'text', 'html' : 'true', 'filterable' : 'false', 'sortable' : 'false' }
-})
 #if ($xcontext.isMainWiki())
-  #set($discard = $columns.add(5, 'wiki'))
-  #set($discard = $columnsProperties.put('wiki', { 'type' : 'text', 'size' : 6 }))
+  #set ($discard = $columns.add(5, 'wiki'))
 #end
-#set($options = {
-   'resultPage': 'Mail.SendMailStatusJSON',
-   'translationPrefix' : 'Mail.SendMailStatus_livetable.',
-   'tagCloud' : false,
-   'selectedColumn' : 'date',
-   'defaultOrder' : 'desc',
-   'outputOnlyHtml': true
+#set ($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+      { 'id': 'date', 'displayer': 'text' },
```

Comment:
I didn't find a way to configure this column to have a working date date filter.

Reply:
WDYM by "working date filter"? The date filter is loading for me when using ``?useLiveData=true``. Sure, it doesn't filter the rows because the source (``Mail.SendMailStatusJSON``) doesn't support filtering on the date column, but that's not different than with the old live table. So you just need to make the date column un-filterable.